integration-test/background/atb.js

```diff
@@ -25,12 +25,9 @@ describe('install workflow', () => {
             // wait for post install page to open
             // if it never does, jasmine timeout will kick in
             while (!postInstallOpened) {
+                const urls = await Promise.all(browser.targets().map(target => target.url()))
```

Comment:
Note: `browser.targets()` returns an array (_not_ a promise) and `target.url()` return a promise (see [docs](https://pptr.dev/#?product=Puppeteer&version=v7.1.0&show=api-browsertarget))